### PR TITLE
Adding some SEO improvements in tags, sitemap and robots

### PR DESCRIPTION
**Pull request by aneudyx**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,19 +1,26 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width,initial-scale=1.0">
```
Comment (masterfermin02) [inline public/index.html]: Can you confirm if removing this tag the responsive continue working ?

File: public/index.html [hunk 2]
```diff
@@ -1,19 +1,26 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (masterfermin02) [inline public/index.html]: Why are we removing this tag ?

File: public/index.html [hunk 3]
```diff
@@ -1,19 +1,26 @@
...
-    <meta property="og:site_name" content="Healty, Org.">
-    <meta name="twitter:image:alt" content="Corona virus en RD">
+    <meta property="og:image" content="/logo-mdb-vue-small.png">
+
+    <!-- Twitter -->
+    <meta property="twitter:card" content="summary_large_image">
+    <meta property="twitter:url" content="https://www.coronavirus-rd.com/">
+    <meta property="twitter:title" content="Estado actual del Coronavirus en REP DOM.">
+    <meta property="twitter:description" content="Esta aplicacion busca llevar a sus usuarios informacion resumida y actualizada por el ministerio de Salud Publica sobre la propagacion del virus COVID-19.">
+    <meta property="twitter:image" content="/logo-mdb-vue-small.png">
```
Comment (masterfermin02) [inline public/index.html]: All of those tag are resuming with this `<meta property="twitter:card" content="summary_large_image">`

File: public/index.html (hunk 1 above)
Comment (masterfermin02) [reply public/index.html]: I just confirmen please don't remove this tags also this tag `<meta charset="utf-8">` to allow the browser knows what charset conversion we are using.

File: public/index.html (hunk 2 above)
Comment (aneudyx) [reply public/index.html]: I removed them by mistake. Will add it again

File: public/index.html (hunk 1 above)
Comment (aneudyx) [reply public/index.html]: This was removed by mistake.

File: public/sitemap.xml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+<!-- created with Free Online Sitemap Generator www.xml-sitemaps.com -->
+
+
+<url>
+  <loc>https://www.coronavirus-rd.com/</loc>
+  <lastmod>2020-03-25T17:14:47+00:00</lastmod>
+</url>
+
+
+</urlset>
```
Comment (masterfermin02) [inline public/sitemap.xml]: Add a line here would be good for github.

**Pull request merged**